README.md

```diff
@@ -926,6 +926,10 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2022.
 - <a name="Xiao22tci"></a>Xiao, Z., Weng, W., Zhang, Y., Xiong, Z.,  
 *[EVA2: Event-Assisted Video Frame Interpolation via Cross-Modal Alignment and Aggregation](https://doi.org/10.1109/TCI.2022.3228747)*,  
 IEEE Trans. Comput. Imag. (TCI), 8:1145-1158, 2022.
+- <a name="Sun23cvpr"></a>Sun, L., Sakaridis, C., Liang, J., Sun, P., Cao, J., Zhang, K., Jiang, Q., Wang, K., Gool, L.,  
+*[Event-Based Frame Interpolation with Ad-hoc Deblurring](https://arxiv.org/abs/2301.05191)*,  
+IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://arxiv.org/pdf/2301.05191.pdf), [Code](https://github.com/AHupuJR/REFID), [Suppl.](https://arxiv.org/pdf/2301.05191.pdf), [Project page](https://ahupujr.github.io/REFID/)
```

Comment:
Congratulations about cvpr.

No need to have the link to arXiv 3 times. One is enough.